react/.eslintrc

```diff
@@ -4,5 +4,8 @@
     "browser": true,
     "es6": true,
     "jest": true
+  },
+  "rules": {
+    "react-hooks/exhaustive-deps": "warn"
```

Comment:
This should not be added.

Reply:
I removed it once I used the hook recommended below, but keep in mind that without this rule set on "warn" eslint will not let the usage of useEffect with empty list on dependencies. So it will help you in general not to use useEffect dependent on some changeable code that will result in memory leak but has a downside for this use case ( sincerely I don't understand completely why it doesn't let you because in the react documentation when it is not deps the useEffect will be an equivalent to componentDidMount from the class component